webapp/sources/rudder/rudder-web/src/main/elm/sources/QuickSearch/View.elm

```diff
@@ -0,0 +1,157 @@
+              span [ class "glyphicon glyphicon-remove", onClick (UpdateSearch "") ] []
+            ]
+          , a [ class "input-group-addon",  id "help-search",  href "/rudder-doc/reference/current/usage/node_management.html#search-nodes",  target "_blank" ] [
+              span [ class "glyphicon glyphicon-question-sign" ] []
+             ]
+          , div [ id "background-fade-quicksearch", onClick Close  ] []
+          , ul [ class "dropdown-menu dropdown-search dropdown-menu-right" ] [
+              li [ class "dropdown-header" ] [
+                div [ class "row no-margin" ] [
+                  div [ class "row",  id "search-tab" ] [
+                    div [ class "filter-search col-xs-12" ] [
+                      div [ class "panel panel-default panel-help" ] [
+                        div [ class "panel-heading heading-search" ] [
+                          div [ class "btn-toolbar", role "toolbar" ] [
+                            div [ class "btn-group group-all", attribute "data-toggle" "buttons" ] [
+                              filterButton model All
+                            ]
+                          , div  [ class "btn-group group-filters", attribute "data-toggle" "buttons" ]
+                              (List.map (filterButton model) allFilters)
+                          ]
+                        ]
+                        , div [ class "panel-body results-content" ] [
+                            div [ class "info-messages" ] [
+                              div [ class "angucomplete-searching", hidden ((String.length model.search ) >= 3 )] [
+                                span [ class "glyphicon glyphicon-exclamation-sign" ] []
+                              , text "The field size must be greater than 2 characters"
+                              ]
+                            , div [ hidden ((String.length model.search ) < 3 )] [ --ng-if="searching || (!searching &amp;&amp; (!results || results.length == 0));" class="ng-scope">
+                                div [ class "angucomplete-searching", hidden (model.state /= Searching) ] [
+                                  text "Searching..."
+                                ]--ng-show="searching &amp;&amp; searchStr.length > 2; " ng-bind="textSearching">Searching...</div>
+                              , div [ class "angucomplete-searching text-danger", hidden ((model.state /= Opened) || (List.length model.results > 0))  ] [-- ng-show="!searching &amp;&amp; (!results || results.length == 0) &amp;&amp; (searchStr.length > 2);">
+                                  span [ class "glyphicon glyphicon-exclamation-sign text-danger" ] []
+                                , text "No results found"
+                                ]
+                              ]
+                              --TODO : look for no result here <!-- ngIf: searching || (!searching && (!results || results.length == 0)); -->
```

Comment:
this one seems to be a real todo

Reply:
but the behavior in test is OK, I have the case well handled: 
![image](https://github.com/Normation/rudder/assets/44649/2350bfb2-7584-44fd-a2ff-e108d67b8eb5)